pom.xml

```diff
@@ -139,7 +139,7 @@
 
     <!-- Required since support for ELS 2.x. Keep in sync with kie-parent or remove when those
           two versions are being updated on the IP BOM.-->
-    <version.com.fasterxml.jackson>2.6.2</version.com.fasterxml.jackson>
+    <version.com.fasterxml.jackson>2.9.5</version.com.fasterxml.jackson>
```

Comment:
The same as above:
There seems to be no reason to keep 2 properties for jackson version in this file now that both versions are aligned.
Please remove one of them and replace occurences with the other.